src/components/app-content-renderer/SimpleSlider.js

```diff
@@ -59,30 +58,27 @@ function SimpleSlider() {
                 </Text>
                 <Text component="p" className="pf-u-flex-grow-1">
                   Try Red Hat OpenShift technologies without setup or
-                  configuration for free with your Red Hat Developer account.
-                </Text>
-                <Flex className="pf-u-flex-direction-row-on-md">
-                  <Text component="p" className="pf-u-pr-sm pf-u-mb-0">
-                    <Link to="/openshift/sandbox">
-                      <Button variant="danger" className="pf-u-pr-sm">
-                        Build apps on the sandbox
-                      </Button>
-                    </Link>
+                  configuration for free with your Red Hat Developer
+                  account.&nbsp; &nbsp;
```

Comment:
Just out of curiosity, why two non breaking spaces? Wouldn't it be better to use `&emsp` or regular `padding-left` with PF's gutter?